### DISCUSSION TITLE
Previous nicknames box is slightly cut when user cover is hidden

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#26088] [Q&A]
Not sure if it's worth an issue, it's just a tiny visual detail. At the end of the day, it can even be right like that (and I guess it can cause too much effort just to solve this thing).

Since users tagged with a group (e.g. `osu! Alumni` in this case) have their name positioned a bit more to the top, the `PreviousUsernamesDisplay` gets cut a bit if I hide the user image cover:

![image](https://github.com/ppy/osu/assets/43073074/30a5f02d-62ae-4682-842e-b093bce2c040)

Of course, this is not an issue if the cover is visible:

![image](https://github.com/ppy/osu/assets/43073074/8862c826-ff49-4e68-9162-8e0a711215b9)

Web handles this a bit better:

![image](https://github.com/ppy/osu/assets/43073074/702d26ec-65db-4550-aa3f-00ac0ad43d67)

**Comment by peppy** [comment]
This is occurring due to depth being reversed for items in this flow:

https://github.com/ppy/osu/blob/ad6af1d9b74302b1d0396baba0fad68f31f88441/osu.Game/Overlays/UserProfileOverlay.cs#L292-L294

Will require some gymnastics to make this work correctly.
